### PR TITLE
[discovery-gce] Upgrade google-oauth-client to 1.33.1

### PR DESCRIPTION
**Pull request by arteam**

### CODE REVIEW

**Comment by arteam** [comment]
@elasticmachine update branch

File: plugins/discovery-gce/build.gradle [hunk 1]
```diff
@@ -7,13 +7,14 @@ esplugin {
 }
 
 versions << [
-  'google': '1.23.0'
+  'google': '1.23.0',
```
Comment (fcofdez) [inline plugins/discovery-gce/build.gradle:10]: is the client forward compatible with the oauth client? Maybe we should upgrade everything? I haven't looked into detail

Comment (arteam) [reply plugins/discovery-gce/build.gradle:10]: Good point, I've submitted #85132

**Comment by fcofdez** [comment]
Thanks for the changes @arteam, let's close this one and I'll review https://github.com/elastic/elasticsearch/pull/85132

**Pull request closed without merging**